notebooks/tensor_inspection.ipynb

```diff
@@ -0,0 +1,1028 @@
+{
```

Comment:
General comment on the readability aspect of this notebook. The formatting is a bit all over the place.

Reply:
This is an issue with Github, not sure if there's a good way around it, but the notebook looks better when used outside github.